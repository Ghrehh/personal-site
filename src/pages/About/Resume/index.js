// @flow
import React from 'react';

import SectionHeading from 'components/SectionHeading';
import ButtonContainer from 'components/ButtonContainer';
import Button from 'components/Button';
import styles from './styles.module.css';

const buttonStyle = additionalStyle => `${styles.button} ${additionalStyle}`;

const Resume = () => (
  <section className={styles.resume}>
    <div className={styles.resumeInner}>
      <SectionHeading className={styles.heading}>CV/RESUME</SectionHeading>

      <p className={styles.body}>
        Please view my CV for a detailed description of my past and current
        work.
      </p>

      <ButtonContainer className={styles.buttonContainer}>
        <Button className={buttonStyle(styles.viewButton)}>View CV</Button>

        <Button className={buttonStyle(styles.downloadButton)}>
          Download CV
        </Button>
      </ButtonContainer>
    </div>
  </section>
);

export default Resume;
