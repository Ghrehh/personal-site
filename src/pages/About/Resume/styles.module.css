.resume {
  padding: 80px 20px 50px;
}

.resumeInner {
  max-width: 500px;
  margin: auto;
  padding: 40px;
  border: 1px solid var(--grey);
  border-radius: 3px;
  background-color: white;
}

.heading {
  text-align: center;
}

.body {
  text-align: center;
}

.buttonContainer {
  text-align: center;
}

.button {
  width: 150px;
}

.viewButton {
  background-color: var(--green);
}

.downloadButton {
  background-color: var(--purple);
}

@media (max-width: 680px) {
  .resume {
    padding: 40px 20px 20px;
  }
}
