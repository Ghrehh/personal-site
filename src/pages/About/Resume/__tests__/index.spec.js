import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import Resume from '../';

describe('Resume', () => {
  let testContext = {};

  beforeEach(() => {
    testContext.renderer = new ShallowRenderer();
    testContext.renderer.render(<Resume />);
    testContext.result = testContext.renderer.getRenderOutput();
  });

  it('renders the component correctly', () => {
    expect(testContext.result).toMatchSnapshot();
  });
});
