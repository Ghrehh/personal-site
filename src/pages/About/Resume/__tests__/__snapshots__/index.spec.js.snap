// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resume renders the component correctly 1`] = `
<section
  className="resume"
>
  <div
    className="resumeInner"
  >
    <Heading
      className="heading"
    >
      CV/RESUME
    </Heading>
    <p
      className="body"
    >
      Please view my CV for a detailed description of my past and current work.
    </p>
    <ButtonContainer
      className="buttonContainer"
    >
      <Button
        className="button viewButton"
      >
        View CV
      </Button>
      <Button
        className="button downloadButton"
      >
        Download CV
      </Button>
    </ButtonContainer>
  </div>
</section>
`;
